### PR TITLE
docs: add listeners intro

### PR DESCRIPTION
**Pull request by lafirest**

### CODE REVIEW

File: zh_CN/getting-started/listeners.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+监听器主要用于配置不同协议的监听端口和相关参数，EMQX 支持配置多个 Listener 以同时监听多个协议或端口。
+
+以下是支持的协议类型：
+
+| 监听器                     | 说明                                             |
+| ------------------------- | ------------------------------------------------------- |
+| TCP Listener              | A listener for MQTT which uses TCP                      |
+| SSL Listener              | A secure listener for MQTT which uses TLS               |
+| Websocket Listener        | A listener for MQTT over WebSockets                     |
```
Comment (tigercl) [inline zh_CN/getting-started/listeners.md:18]:
```suggestion
| Websocket Listener        | A listener for MQTT over WebSocket                     |
```

File: zh_CN/getting-started/listeners.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+| Secure Websocket Listener | A secure listener for MQTT over secure WebSockets (TLS) |
```
Comment (tigercl) [inline zh_CN/getting-started/listeners.md:19]:
```suggestion
| Secure Websocket Listener | A secure listener for MQTT over secure WebSocket (TLS) |
```

File: zh_CN/getting-started/listeners.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+| QUIC Listener             | A secure listener for MQTT over secure WebSockets       |
```
Comment (tigercl) [inline zh_CN/getting-started/listeners.md:20]:
```suggestion
| QUIC Listener             | A secure listener for MQTT over secure WebSocket       |
```

File: zh_CN/getting-started/listeners.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+| 8083            | MQTT/WS protocol port   |
+| 8084            | MQTT/WSS protocol port  |
+
+<a id="org67b1907"></a>
+
+### 通过配置进行添加
+
+监听器配置项的命名规则为 `listener.<Protocol>.<Listener Name>.xxx`， `<Protocol>` 即 Listener 使用的协议。 `<Listener Name>` 可以随意命名，但建议是全小写的英文单词， `xxx` 则是具体的配置项。不同协议的 Listener 的 `<Listener Name>` 可以重复。
+
+由于默认配置的存在，我们能够非常快速地展示如何添加新的监听器，以 TCP 监听器为例，我们只需要在 `emqx.conf` 中添加以下一条配置即可：
```
Comment (tigercl) [inline zh_CN/getting-started/listeners.md]:
```suggestion
由于默认配置的存在，我们能够非常快速地添加新的监听器，以 TCP 监听器为例，我们只需要在 `emqx.conf` 中添加以下一条配置即可：
```

File: zh_CN/getting-started/listeners.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+删除:
+
+```
+curl -X 'DELETE' \
+     'http://127.0.0.1:18083/api/v5/listeners/tcp%3Ademo' \
+     -H 'accept: */*'
+```
+
+在 API 文档中有，有更多和更详细的监听器 API 信息，这里就不再一一列举了。
```
Comment (tigercl) [inline zh_CN/getting-started/listeners.md]:
```suggestion
更详细的监听器 API 信息，请参见 [API 文档](xxx.md)
```
文档里面尽量不要出现类似于博客的写法。

File: zh_CN/getting-started/listeners.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+### 使用 API 进行操作
+
+监听器除了支持使用配置文件进行设置外，还支持通过 HTTP API 进行添加、删除、修改、启动、停止等操作，例如：
+
+添加一个名为 `demo` 的监听器:
+
+
+```
+curl -X 'POST' \
+     'http://127.0.0.1:18083/api/v5/listeners/tcp%3Ademo' \
```
Comment (tigercl) [inline zh_CN/getting-started/listeners.md]: 这里需要解释一下为什么是 `tcp%3Ademo`

File: en_US/getting-started/listeners.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+| 1883            | MQTT/TCP protocol port  |
+| 8083            | MQTT/WS protocol port   |
+| 8084            | MQTT/WSS protocol port  |
+
+<a id="org69700c5"></a>
+
+## Quick Start
+
+
+<a id="org67b1907"></a>
```
Comment (wivwiv) [inline en_US/getting-started/listeners.md]: What is a tag used for?

File: zh_CN/getting-started/listeners.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+| Websocket Listener        | A listener for MQTT over WebSockets                     |
+| Secure Websocket Listener | A secure listener for MQTT over secure WebSockets (TLS) |
+| QUIC Listener             | A secure listener for MQTT over secure WebSockets       |
+
+EMQX 默认提供 4 个监听器，它们将占用以下端口：
+
+| 端口             | 说明                    |
+| ----------------| ----------------------- |
+| 8883            | MQTT/SSL protocol port  |
+| 1883            | MQTT/TCP protocol port  |
```
Comment (wivwiv) [inline zh_CN/getting-started/listeners.md:21]: To be more strict, 1883 should be at the top of the list

File: zh_CN/getting-started/listeners.md (hunk 6 above)
Comment (wivwiv) [reply zh_CN/getting-started/listeners.md]: 先给出 API 文档，再给出示例。更多 API 可以 Link 到 API 章节。

```
POST http://127.0.0.1:1883/api/v5/listeners/{type}:{name}
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: zh_CN/getting-started/listeners.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
+# 监听器
+
+## 简介
+
+监听器主要用于配置不同协议的监听端口和相关参数，EMQX 支持配置多个 Listener 以同时监听多个协议或端口。
```
Comment (hjianbo) [inline zh_CN/getting-started/listeners.md]:
```suggestion
监听器主要用于配置不同协议的监听端口和相关参数，EMQX 支持配置多个监听器以同时监听多个协议或端口。
```

**Review by hjianbo — DISMISSED** [review]

**Review by hjianbo — APPROVED** [review]
Look goods to me. let's merge it as soon as possible

**Pull request merged**